lib/node_modules/@stdlib/repl/lib/sort_fuzzy_completions.js

```diff
@@ -0,0 +1,66 @@
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2024 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+'use strict';
+
+// FUNCTIONS //
+
+/**
+* Callback to sort the results array in descending order of score (lexicographically if scores are equal).
+*
+* @private
+* @param {Object} a - first object representing a completion's fuzzy score
+* @param {Object} b - second object representing a completion's fuzzy score
+* @returns {number} - comparison result for array sorting
+*/
+function sortClbk( a, b ) {
+	if ( b.score === a.score ) {
+		// If scores are equal, sort lexicographically...
+		return a.completion.localeCompare( b.completion );
+	}
+	// Sort by score:
+	return b.score - a.score;
+}
+
+
+// MAIN //
+
+/**
+* Sorts an array of completion objects and returns an array of sorted completion strings.
```

Comment:
Might be good to document that the input argument array is mutated. E.g.,  

```suggestion
* Sorts an array of completion objects in-place and returns an array of sorted completion strings.
```